client/src/constants/routes.js

```diff
@@ -10,6 +10,7 @@ export const Route = Object.freeze({
     // Shared routes.
     Root: '/',
 
+    DashboardB: '/dashboard',
```

Comment:
Typo here? `DashboardB`

Reply:
Ah I see there's another `Dashboard`, why are there two? I can see that being confusing

Reply:
Temporary page for now. Not a typo. Yes, it is confusing and will change once we know where its heading.